gui/default/syncthing/core/syncthingController.js

```diff
@@ -271,19 +272,20 @@ angular.module('syncthing.core')
 
             if (arg.data.added) {
                 arg.data.added.forEach(function (rejected) {
-                    var offeringDevice = {
+                    var observedFolder = {
```

Comment:
I can see why you might want to rename this, matching the DB's `ObservedFolder` type.  But for consistency in the GUI code, I would keep the `offeringDevice` naming, because the rejection is always specific to the device, and `observedFolder` next to `pendingFolder` is more confusing IMO.

Just a suggestion though, I'm okay with the rename if it's clearer to you this way.

Reply:
You're right about consistency, and it's unrelated to this PR - I'll revert it.

The name was very confusing to me though: The object contains no device info, but e.g. `label`, which doesn't label a device, but a folder. To me  `observedFolder` is better, or maybe `deviceFolder` would also make sense.

Reply:
I totally understand the confusion.  `offerDetails` maybe?  That would make line 288 readable again.